recipes/mne-python/construct.yaml

```diff
@@ -96,12 +97,13 @@ specs:
   - python-neo =0.13.0
   - pandas =2.2.0
   # BLAS
-  - openblas =0.3.25
+  - openblas =0.3.26
   - libblas =3.9.0=*openblas
   - jupyter =1.0.0
   - jupyterlab =4.1.2
   - ipykernel =6.29.2
-  - nb_conda_kernels =2.3.1
+  # https://github.com/Anaconda-Platform/nb_conda_kernels/issues/230
+  # - nb_conda_kernels =2.3.1
```

Comment:
Actually I don't think we need this anyway, it's only required to select kernels from a different conda environment. I believe we should be fine without this in the installer